.github/workflows/prepare-release.yml

```diff
@@ -0,0 +1,44 @@
+name: Prepare Release
+
+on:
+  workflow_dispatch:
+env:
+  REALM_CI: true
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Update Changelog
+        id: update-changelog
+        uses: realm/ci-actions/update-changelog@dac063f8ebd86c201590d07536d6f56f86f66ba9
+        with:
+          changelog: ${{ github.workspace }}/CHANGELOG.md
+
+      - name: Update pubspec.yaml
+        uses: jacobtomlinson/gha-find-replace@b76729678e8d52dadb12e0e16454a93e301a919d #! 2.0.0
+        with:
+          find: 'version: .*'
+          replace: 'version: ${{ steps.update-changelog.outputs.new-version }}+alpha'
```

Comment:
So `+alpha` is hardcoded in the action.. I guess that is okay

Reply:
Yeah, that's only temporary until we move to 1.x at which point we can drop the +alpha.